shared/menubar/remote-container.desktop.js

```diff
@@ -75,9 +79,8 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   return {
     ...stateProps,
     ...dispatchProps,
-    refreshUserFileEdits: dispatchProps.refreshUserFileEdits,
+    onRetrySync: dispatchProps._onRetrySync(stateProps.username),
```

Comment:
Did you mean to remove `refreshUserFileEdits and waitForKbfsDaemon`?

Reply:
they're already included with `...dispatchProps`.